package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "jsgantt-improved",
+  "version": "2.0.2",
+  "description": "jsgantt-improved",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc",
+    "watch": "onchange 'src/**' '*.*' -- npm run build",
+    "watch:dist": "onchange 'src/**' '*.ts' -- npm run dist",
+    "watch:test": "onchange 'src/**/*.ts' '*.ts' -- npm run test",
+    "test": "node node_modules/.bin/protractor protractor.conf.js",
+    "test-unit": "mocha -r ts-node/register test/**.ts",
+    "browserify": "browserify dist/index.js --standalone JSGantt > jsgantt.js",
+    "dist": "npm run build && npm run browserify && cp src/jsgantt.css dist/ && cp jsgantt.js dist/ && cp jsgantt.js demo/ && cp src/jsgantt.css demo/ && echo 'DIST finished'",
+    "publish-npm": "npm run dist && npm publish",
+    "demo": "nodemon server.js",
+    "demo-full": "npm run dist && npm run demo",
+    "e2e-prepare": "npm i -g webdriver-manager && webdriver-manager update && ./node_modules/protractor/node_modules/webdriver-manager/bin/webdriver-manager update"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mariohmol/jsgantt-improved"
+  },
+  "author": "Eduardo Rodrigues, Ricardo Cardoso, Mario Mol <mariohmol@gmail.com>",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/mariohmol/jsgantt-improved/issues"
+  },
+  "homepage": "https://github.com/mariohmol/jsgantt-improved#readme",
+  "dependencies": {},
+  "devDependencies": {
+    "@types/chai": "^4.1.5",
+    "@types/jasmine": "^3.3.0",
+    "chai": "^4.1.2",
```

Comment:
I had to install `browserify` and `nodemon` because didn't have them globally.

Reply:
included it on readme